core/src/main/scala/com/rackspace/com/papi/components/checker/Validator.scala

```diff
@@ -98,8 +107,8 @@ class Validator private (private val _name : String, val startStep : Step, val c
 
 
   private val platformMBeanServer = ManagementFactory.getPlatformMBeanServer()
-  private val objectName = new ObjectName("\"com.rackspace.com.papi.components.checker\":type=\"Validator\",scope=\""+
-                                 name+"\",name=\"checker\"")
+  private val objectName = new ObjectName("com.rackspace.com.papi.components.checker:type=Validator,scope="+
+                                 name+",name=checker")
```

Comment:
These are just the key names to look up in the JMX Registry. So yes, quotes are valid characters and they were showing up in the actual names. From what I can tell, the intent was to identify make them Strings. However they are all Strings and the quotes were included in the String.

I updated it for consistency with all other registered JMX items which did not have them.